### ISSUE TITLE
az sample for sp is invalid

### ISSUE THREAD

**Issue by mifurm** [microsoft/openshift-origin#112]
in prereq. you have a sample to create SPN in Azure by: az ad sp create-for-rbac -n openshiftcloudprovider --password Pass@word1 --role contributor --scopes /subscriptions/555a123b-1234-5ccc-defgh-6789abcdef01. However, since AZ CLI 2.0.68 (https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest) --password is not longer valid.

**Comment by satya2048** [comment]
Hi mifurm you could try this
1.az ad sp create-for-rbac --name serviceprincipal
2.az role assignment create --assignee appidfromsp --scope yourscope --role Contributor

**Comment by mifurm** [comment]
Yes! You are right, That look perfect and works for me. I left the comment so you can update the docs. I can create PR if needed :)
